src/components/BookstoreComponentsStyles/card.js

```diff
@@ -1,34 +1,48 @@
+import { useSelector, useDispatch } from 'react-redux';
 import Button from './button';
+import { removeBook } from '../../redux/books/bookSlice';
 
-const Card = ({ value }) => (
-  value.map((cur) => (
-    <div className="card-container" key={0}>
-      <div className="card-details">
-        <div className="min1">
-          <p>{cur.Catogery}</p>
-          <h3>{cur.BookTitle}</h3>
-          <p>{cur.Author}</p>
-          <div className="btns-func">
-            <a href="/">Comments</a>
-            <a href="/">Remove</a>
-            <a href="/">Edit</a>
+const Card = () => {
+  const Dispatch = useDispatch();
+  const Booklists = useSelector((state) => state.book.Booklist);
+  return (
+    Booklists.map((cur) => (
+      <div className="card-container" key={cur.item_id}>
+        <div className="card-details">
+          <div className="min1">
+            <p>{cur.catogery}</p>
+            <h3>{cur.title}</h3>
+            <p>{cur.author}</p>
+            <div className="btns-func">
+              <a href="/">Comments</a>
+              <a
+                href="/"
+                onClick={(e) => {
+                  Dispatch(removeBook(cur.item_id));
+                  e.preventDefault();
+                }}
+              >
+                Remove
+              </a>
```

Comment:
[Optional] Nice job implementing the remove functionality✨. However, it would be better to use a `<button>` element here instead of an `<a>` tag. This way, you don't need to add the href attribute and add `e.preventDefault()`. I highly suggest you update this👍